app/theme/client/main.css

```diff
@@ -58,3 +58,344 @@
 
 /* RTL */
 @import 'imports/general/rtl.css';
+
+/* Visitor History CSS */
+.inner-addon{
+	position: relative;
+	left: 24px;
+	width: 345px;
+	height: 40px;
+	background: #ffffff;
+	border: 2px solid #CBCED1;
+	box-sizing: border-box;
+	border-radius: 2px;
+}
+.inner-addon .glyphicon{
+	position: absolute;
+	top:12.5%;
+	bottom:12.5%;
+	color: #2F343D;
+	padding: 10px;
+	pointer-events: none;
+	
+}
+.searchresults{
+    margin-top:7%;
+}
+.search-li{
+    display: block;
+    padding-bottom: 3%;
+    position: relative;
+    height: 65px;
+    left:24px;
+    background-color: #ffffff;
+}
+.closed-div{
+    position: relative;
+    top:20px;
+}
+.closed-div-p{
+    background: #F5455C;
+    border-radius: 2px;
+    width: 100%;
+    text-align: center;
+    line-height: 0.15em;
+    font-size: 10px;
+    margin:10px 0 15px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 1 tab (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5265440)

Reply:
done please check